comms/core/src/connectivity/test.rs

```diff
@@ -392,7 +392,7 @@ async fn pool_management() {
     // Drop all connections references
     for mut conn in connections {
         if conn != important_connection {
-            assert_eq!(conn.handle_count(), 2);
```

Comment:
The problem is the `let _result` that was added on line 384 - previously I explicitly dropped the events by writing `let _ =` but `let _result = ` drops at the end of the function. To fix this test, and to keep clippy happy, delete `let _result =` and the handle counts will be 2 again